Code/Engine/Foundation/Math/Implementation/AllClasses_inl.h

```diff
@@ -324,3 +324,30 @@ ezResult ezMat4Template<Type>::Invert(Type fEpsilon)
   *this = Inverse;
   return EZ_SUCCESS;
 }
+
+//////////////////////////////////////////////////////////////////////////
+
+// static
+template <typename T>
+bool ezComparisonOperator::Compare(ezComparisonOperator::Enum cmp, const T& a, const T& b)
+{
+  switch (cmp)
```

Comment:
this makes my head hurt, do we have a unit test that validates this is correct, e.g. for floats ?

Reply:
added a unit test